contrib/docker/data/etc_init.d_artifactory_orig_2015-07-16_0912UTC.sh

```diff
@@ -0,0 +1,287 @@
+#!/bin/bash
```

Comment:
How about a note to where this file could be found in JFrog's world in your script?  I'm not sure it belongs in VCS here.